jedi/inference/compiled/access.py

```diff
@@ -487,7 +487,7 @@ def needs_type_completions(self):
     def _annotation_to_str(self, annotation):
         if isinstance(annotation, type):
             return str(annotation.__name__)
-        return str(annotation)
+        return str(annotation).replace('typing.', '')
```

Comment:
This is wrong, `my_custom_module.typing.Something` is now `my_custom_module.Something`.

Reply:
I missed the fact that `inspect` first check if symbol came from `typing`. Fixed.